### PR TITLE
[RM Ch3] Redefine NFVI Management Software

### LINKED ISSUE #559: [RM] Chapter 3 - clarification of NFVI v VIM
The diagram in section 3.1 ([link](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter03.md#31-model)) suggests that the "management software" is part of the NFVI. Is this not the VIM, and therefore not within NFVI but alongside it?

In addition the following definition seems confused:
> NFVI Management Software: This consists of both the host Operating System (OS) responsible for managing the physical infrastructure resources as well as the virtualization/containerization technology which, on request, dynamically allocates hardware components and exposes them as virtual resources.

I would propose that the word "Management" is removed from the definition above so it reads "NFVI Software" - this would match later use of the term in the profiles sections.

In addition, section 3.3 ([link](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter03.md#33-management-software)) describes what I'd see as a VIM, but this doesn't match the description in section 3.1.

In summary, I would propose the following changes:

1. Change the diagram in 3.1 to show NFVI Management as being separate to NFVI
1. Remove the word "Management" from the definition in section 3.1
1. Add a section (3.5) that describes the "NFVI Software" component (this isn't included yet)
1. Change the title and wording of section 3.3 to refer to VIM (or another similar term that makes it clear it's the management of the virtual and physical resources)

### PR DESCRIPTION
**Pull request by tomkivlin**
Update definition to include VIM features. Re-order the functionalities to match the subsequent L1 sections. Update name of section 3.3 to match functionality list.

Fixes #559

### CODE REVIEW

**Comment by tomkivlin** [comment]
Some background to the changes I've made:
- Firstly, I've updated the section heading for 3.3 to "NFVI Management Software" to match the bullet list and diagram
- I've re-ordered the bullet list to match the order of the sections 3.2, 3.3 and 3.4
- In line with @BernardTsai-DT's comments in #559 wrt NFVI software and VIM being quite tightly coupled in real life (I agree), I've broadened the wording of "NFVI Management Software" to include not only Host OS and hypervisor/containerisation but also the logical management capabilities around tenants, etc. and all of that being accessible via API

Hope that makes sense.

**Comment by rabi-abdel** [comment]
@BernardTsai-DT @markshostak @kedmison can you please review?

File: doc/ref_model/chapters/chapter03.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ The abstraction model for the NFVI makes use of the following layers (only the v
...
 - **Virtual Infrastructure Resources:** These are all the infrastructure resources (compute, storage and networks) which the NFVI provides to the workloads such as VNFs/CNFs. These virtual resources can be managed by the tenants and tenant workloads directly or indirectly via an application programming interface (API).
+- **NFVI Management Software:** This consists of the software components that manage the physical and virtual resources and make those management capabilities accessible via one or more APIs. The host Operating System (OS) is responsible for managing the physical infrastructure resources and abstracting them from processes running within the OS. Virtualisation / containerisation technology dynamically allocates these abstracted hardware components and exposes them as virtual resources. Additional software is responsible for the management of logical constructs such as tenants, tenant workloads, resource catalogues, identities, access controls, security policies, etc.
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter03.md]: The sentence about "Additional software is responsible for the management of logical constructs such as tenants, etc" might be confusing. What kind of tenants, catalogues etc. is it referring to in this context?

Comment (tomkivlin) [reply doc/ref_model/chapters/chapter03.md]: VIM/NFVI tenants, VM catalogues, and so on - essentially the features that you'd expect from a VIM, but that aren't included in the rest of the description for "NFVI Management Software".

Comment (ulikleber) [reply doc/ref_model/chapters/chapter03.md]: Yes. That's explained in 3.3 below. The short sentence here could be confused with the VIM which has similar things. But just 2 c.

Comment (tomkivlin) [reply doc/ref_model/chapters/chapter03.md]: Yeah the purpose of this PR is to match the description here with the content in 3.3 - so we are deliberately including VIM functionality in this description here. Before this PR, they were different and had been causing confusion.

**Comment by rabiabdel** [comment]
This PR will be marked as "idle" and it will be "force merged" if no more reviews/approvals received by then.

**Pull request merged**